src/components/AccordionComponent/Accordion.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode, useState } from 'react';
+import ClassTrackerItems from './ClassTrackerItems';
+
+type ClassTrackerItemsType = {
+  title: string;
+  content: ReactNode;
+};
+
```

Comment:
This looks really good! But for consistency, Janice uses interface for props and she declares array this way `string[]` instead of `Array<string>`
So we can just add interface for the props here.
```Typescript
interface AccordionProps {
  items: ClassTrackerItemsType[]
}
```

Reply:
Hey, Yes! it makes sense, I will update that.